pkg/service/active_directory_membership_service.go

```diff
@@ -119,7 +119,7 @@ func (svc *ActiveDirectoryMembershipService) DeleteActiveDirectoryMembership(id
 func (svc *ActiveDirectoryMembershipService) RemoveActiveDirectoryMembership(id string, user string, password string, force *bool) error {
 
 	if len(id) == 0 || len(user) == 0 || len(password) == 0 {
-		return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v, password: %v ", id, user, &password)
+		return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v, ", id, user)
```

Comment:
nit: 	Remove the comma/space at the end of the message.

`return fmt.Errorf("RemoveActiveDirectoryMembership: invalid parameter specified id: %v, user: %v", id, user)`

Reply:
thanks @suneeth51 .